app/src/main/java/com/stl/skipthelibrary/MyBooksActivity.java

```diff
@@ -11,19 +15,34 @@
 import androidx.recyclerview.widget.RecyclerView;
 
 public class MyBooksActivity extends AppCompatActivity {
+    private static final String TAG = MyBooksActivity.class.getSimpleName();
+    public static final int ADD = 1;
+
     ArrayList<Book> books = new ArrayList<Book>();
     RecyclerView recyclerView;
+    FloatingActionButton addBookButton;
+    Context mContext;
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_ownerbooks);
-        recyclerView = findViewById(R.id.ownerBooksRecyclerView);
+        recyclerView = (RecyclerView) findViewById(R.id.ownerBooksRecyclerView);
+        addBookButton = (FloatingActionButton) findViewById(R.id.addBookButton);
+        mContext = getApplicationContext();
 
         BottomNavigationView navigation = findViewById(R.id.bottom_navigation);
         navigation.setOnNavigationItemSelectedListener(new NavigationHandler(this));
         navigation.setSelectedItemId(R.id.my_books);
 
+        addBookButton.setOnClickListener(new View.OnClickListener() {
```

Comment:
Instead of an onClickListener could we just have this as a separate method specified in the xml? That would make our onCreate less hectic

Reply:
The method will probably still be defined somewhere in this file, so personally I think this is ok? I guess it may get even busier as this activity grows tho. This is also something that could be refactored later

Reply:
I personally like onClickListeners in the code, it's easier to find the correct method @tymoorej @gillianpierce . Is that cool for now?

Reply:
Yeah I'm okay with that